docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -213,6 +215,60 @@ inside the container, so you can simply run:
 
 5. Stop and remove container
 
+.. code-block:: bash
+
+    docker stop pelux-build
+    docker rm pelux-build
+
+Particular layer build procedure:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Clone the pelux-manifests git repository with submodule
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/Pelagicore/pelux-manifests.git
+    cd pelux-manifests
+
+2. Copy layer directory to pelux-manifests
+
+.. code-block:: bash
+
+    cp -r ../<LAYER_NAME> .
+
+3. Build and run Docker container
+
+.. code-block:: bash
+
+    docker build --build-arg userid=$(id -u) --build-arg groupid=$(id -g) -t pelux .
```

Comment:
Is there a need to build the container again? I would assume the container can be reused if it has been built previously. Perhaps this shall be mentioned?

Reply:
note added